src/main/scala/cromwell/binding/values/WdlString.scala

```diff
@@ -4,7 +4,13 @@ import cromwell.binding.types.WdlStringType
 import org.apache.commons.lang3.StringEscapeUtils
 import scala.util.{Success, Try}
 
-case class WdlString(value: String) extends WdlPrimitive {
+trait WdlStringLike {
+  def value: String
+  def isGcsUrl: Boolean = value.startsWith("gs://")
```

Comment:
JES engine functions care because you can do `read_string("gs://bucket/object")`  If you have a better way to express this, I'm all ears